pyproject.toml

```diff
@@ -58,9 +61,6 @@ inline-quotes = "single"
 quote-style = "preserve"
 indent-style = "space"
 
-[tool.ruff.flake8-quotes]
-inline-quotes = 'single'
```

Comment:
I think Benjamin decided on single quotes for the project, unless I'm mistaken.

Reply:
This PR was behind the current development branch, but I manually resolved the conflicts on my end.

Reply:
This was just removing a duplicate section.

Reply:
Double checked the merge. The removed section is deprecated and we already have this
```toml
[tool.ruff.lint.flake8-quotes]
inline-quotes = "single"
```